libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFManagedPreferences.m

```diff
@@ -94,6 +95,13 @@ - (id)init {
     return self;
 }
 
+- (void)initializeManagedPreferences {
+    self.rawPreferences = [[NSUserDefaults msdkUserDefaults] dictionaryForKey:kManagedConfigurationKey];
+    if (self.rawPreferences) {
```

Comment:
Why don't we just move this block to `[SalesforceSDKManager configureManagedSettings]`, and avoid the deadlock altogether.  This is ultimately for a feature contained in `SalesforceSDKManager` anyway.